index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<h1 id="title">Build a Survey</h1>
```

Comment:
There should be head and body elements, with the head containing elements such as the title, character encoding (meta element) and a link to a css stylesheet.